bigchaindb/db/utils.py

```diff
@@ -67,6 +67,18 @@ def _connect(self):
                     time.sleep(2**i)
 
 
+def get_backend():
+    '''Get a backend instance.'''
+
+    from bigchaindb.db.backends import rethinkdb
+
+    # NOTE: this function will be re-implemented when we have real
+    # multiple backends to support. Right now it returns the RethinkDB one.
+    return rethinkdb.RethinkDBBackend(host=bigchaindb.config['database']['host'],
```

Comment:
These parameters should be passed to `get_backend` and not read from the config or else if someone instantiates Bigchain with custom parameters they will not be passed to `get_backend`.